walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -159,39 +161,30 @@
   private void findAndAddGapsInOccupiedTimeRanges(long requestedDuration,
       List<TimePoint> orderedTimePoints,
       Collection<TimeRange> availableTimeRanges) {
-    int index = 0;
-    int startCounter = 0;
-    int endCounter = 0;
+    int index = 1;
+    int unendedTimeRange = 1;
 
     while (index < orderedTimePoints.size()) {
```

Comment:
This could be a for loop?

Reply:
And don't necessarily need to skip first index either.  Can just be handled naturally (see my comments below)